apps/server/src/modules/authorization/domain/service/authorization.service.ts

```diff
@@ -1,11 +1,11 @@
 import { Injectable, UnauthorizedException } from '@nestjs/common';
-import { AuthorizableObject } from '@shared/domain/domain-object';
-import { BaseDO } from '@shared/domain/domainobject';
-import { User } from '@shared/domain/entity';
-import { EntityId } from '@shared/domain/types';
-import { UserRepo } from '@shared/repo/user';
+import type { AuthorizableObject } from '@shared/domain/domain-object';
+import type { BaseDO } from '@shared/domain/domainobject/base.do';
+import type { User } from '@shared/domain/entity/user.entity';
+import type { EntityId } from '@shared/domain/types/entity-id';
```

Comment:
remove

Reply:
I suppose the "remove" is on the `'@shared/domain/types/entity-id'` and not the other imports?